config/routes.rb

```diff
@@ -11,9 +11,13 @@
 
   devise_for :users
   concern :exportable, Blacklight::Routes::Exportable.new
+        
+    concern :hierarchy, Arclight::Routes::Hierarchy.new
```

Comment:
Same as the next line
```suggestion
```

Reply:
We might want to record this as a flaw in the arclight:update generator.